pleskdistup/actions/mounts.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2023-2024. WebPros International GmbH. All rights reserved.
+
+import os
+
+from pleskdistup.common import action, mounts
+
```

Comment:
I think we can move `get_fstab_configuration_misorderings()` to here. Is there an obstacle that I can't see?

Reply:
We will have to redo tests imports, which I would like to avoid. So I propose to keep it as is